hazelcast-client-sample.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
I think a sample should be short. Its purpose is to convey what the configuration would look like. This is full configuration and it already exists in the repo as `hazelcast-client-full.json`